app_dart/lib/src/request_handlers/get_green_commits.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:cocoon_service/src/model/appengine/stage.dart';
+import 'package:cocoon_service/src/model/appengine/task.dart';
+import 'package:github/github.dart';
+import 'package:meta/meta.dart';
+
+import '../request_handling/body.dart';
+import '../request_handling/request_handler.dart';
+import '../service/build_status_provider.dart';
+import '../service/config.dart';
+import '../service/datastore.dart';
+
+/// Returns a list of the commit shas that had green runs.
```

Comment:
Take advantage of the dart docs format

```suggestion
/// Returns [List<String>] of commit shas that had all passing tests.
```